assets/chat/js/const.js

```diff
@@ -48,4 +48,11 @@ function isKeyCode(e, code){
     return getKeyCode(e) === code
 }
 
-export {KEYCODES,DATE_FORMATS,isKeyCode,getKeyCode}
\ No newline at end of file
+// emote:key; css class to apply
+const generify_options = {
```

Comment:
Should probably be `GENERIFY_OPTIONS` to match the casing of other constants in this file.